### ISSUE TITLE
Taco Master (PCSE00771)

### ISSUE THREAD

**Issue by Yohoki** [r0ah/vitacheat#58]
~~~~
# Title: Taco Master
# Region: US
# Version: 1.00
# Type: NoNpDRM
# Code Author: Yohoki
# Source: https://gbatemp.net/threads/vitacheat-finalcheat-database.485343/page-116#post-8538407
# ID: PCSE00771

_V0 Instant Gold - Finish 1 Order to Win
$3202 851DFFE8 00000010
$3200 00000000 00000068
$3300 00000000 00000001 #set gold medal number to 1
#  --------------------
$3202 851DFFE8 00000010
$3200 00000000 0000006C
$3300 00000000 00000001 #set bronze and silver to 1
#  --------------------
$3202 851DFFE8 00000010
$3200 00000000 FFFFFFF0
$3300 00000000 839FFFF0 #add fake pointer to static address 839FFFF0
#  --------------------
$0200 839FFFF0 851DFFE8 #add extra tier to main pointer to use pointer MOV
#  --------------------
$8203 851DFFE8 00000010
$8200 00000000 FFFFFFF0
$8200 00000000 00000004
$8800 00000000 00000000
$8603 839FFFF0 00000000
$8600 00000000 00000010
$8600 00000000 00000070
$8900 00000000 00000000 #pointer MOV number of orders to static address
#  --------------------
$D801 839FFFF4 00000001 #check if static address is bigger than 1
#  -------------------- #if (Yes) {then
$7002 851DFFE8 00000010
$7000 00000000 00000070
$7702 00000000 00000000
$0003 00000004 00000000 #set important values to 0
~~~~

**Comment by r0ah** [comment]
Thanks, added to the db. So you were experimenting huh? IDK, I like to keep it simple but , hey that's just me. Feel free to do whatever you like.

Also, you might wanna look into this code made by xJam.es: https://github.com/r0ah/vitacheat/blob/master/db/PCSB00769.psv

Might help with your experiment.

**Comment by Yohoki** [comment]
Lmao yes, that looks pretty similar to what I did. I wish the notes were a bit more detailed, since I don't have the game to actually see what he's doing. But, it looks like he's made a couple pointers with the "Always On" code. I think these are where the game's original pointers are stored, so they are no longer random? Hard to tell. But then the last code makes another new pointer and uses it pretty much exactly how I used it. Looks like he's done it before I had.

I'm definitely not the first to do it. Using code caves is a pretty common thing in PC game hacking, with Cheat Engine having an automatic method for finding code caves and a tutorial that teaches you to do it for assembly injection. The methods are out there to learn to do it, but I haven't seen any for Vita. Having more ways to do it is definitely a good thing though. Especially since it was impossible to do otherwise in this case. There's no "P.Conditional" like there is with "P.Compress" and "P.MOV"
